Unix/build.mak

```diff
@@ -161,10 +161,15 @@ distclean:
 ##==============================================================================
 
 ifdef BUILD_TESTS
+
+.PHONY: tests
+
 tests:
-	( LD_LIBRARY_PATH=$(LIBDIR); export LD_LIBRARY_PATH; DYLD_FALLBACK_LIBRARY_PATH=$(LIBDIR); export DYLD_FALLBACK_LIBRARY_PATH; DYLD_LIBRARY_PATH=$(LIBDIR); export DYLD_LIBRARY_PATH; $(BINDIR)/nits -file:$(TMPDIR)/nitsargs.txt )
+	bash -c "trap 'trap - SIGINT SIGTERM ERR; ./tools/auth_tests_cleanup.sh' SIGINT SIGTERM ERR; ./tools/make_tests.sh $(LIBDIR) $(BINDIR) $(TMPDIR)"
```

Comment:
I observed (in a later change) that `kill -SIGTERM` doesn't work on Solaris. Does `trap - SIGTERM` work there reliably?